SR2024/2023-11-05-srawn-05.md

```diff
@@ -0,0 +1,71 @@
+The Student Robotics (Almost) Weekly Newsletter is a (roughly) fortnightly newsletter which summarises the goings‐on across Student Robotics. **Emboldened** items are ones for which additional help has been requested.
+
+## The Competition Team Needs You!
+
+As we gear up for the competition we're once again looking at assigning Area Owners and Deputies. To help us organise the competition, we've split up the necessary tasks into different areas each with an Owner and Deputies. They are responsible for ensuring tasks are completed (not necessarily doing all the tasks on their own!). We have Owners for each of the areas, listed below, and are looking for people that may be interested in becoming a Deputy.
```

Comment:
Suggestion: Perhaps worth linking to "area owners" in the runbook? https://studentrobotics.org/runbook/programme/area-owners/

Reply:
👍 [1cc68b5](https://github.com/srobo/srawn/pull/68/commits/1cc68b5f301af9ee6ce29eb61b117ccd0b4820ff)